include/hse/hse.h

```diff
@@ -110,6 +110,17 @@ hse_err_to_errno(hse_err_t err);
 HSE_EXPORT size_t
 hse_strerror(hse_err_t err, char *buf, size_t buf_len);
 
+/** @brief Return an hse_err_t values error context
+ *
+ * The hse_err_t scalar value @p err is translated into its error context equivalent.
+ * This function is thread safe.
+ * @param err: Error value returned from an HSE API function.
+ *
+ * @returns The error's context.
+ */
+HSE_EXPORT uint16_t
+hse_err_to_ctx(hse_err_t err);
```

Comment:
An `int` would be a better return type.  Caller doesn't need to know that it came a 16-bit field.  Or switch to `enum`.

Reply:
enum!